### PR TITLE
Added recipe for defusedxml

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/defusedxml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+test:
+  imports:
+    - defusedxml
+
+about:
+  home: https://bitbucket.org/tiran/defusedxml
+  license: PSFL 2.0
+  license_file: LICENSE
+  license_family: Proprietary
```
Comment (jakirkham) [inline recipes/defusedxml/meta.yaml]: That doesn't seem right. Isn't PSF open source? 😕

Comment (pmlandwehr) [inline recipes/defusedxml/meta.yaml]: Going by [`conda-build/metadata.py`](https://github.com/conda/conda-build/blob/master/conda_build/metadata.py), it's `PSF`

**Comment by jakirkham** [comment]
Toggling to re-run CIs.

Note to self: The linter correctly handle this case and did not raise a merge conflict thanks to PR ( https://github.com/conda-forge/conda-forge-webservices/pull/68 ).

File: recipes/defusedxml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - defusedxml
+
+about:
+  home: https://bitbucket.org/tiran/defusedxml
+  license: PSFL 2.0
```
Comment (jakirkham) [inline recipes/defusedxml/meta.yaml]: `PSF` is fine. `L` can be dropped.

**Comment by pmlandwehr** [comment]
(ping @ocefpaf for merge)

**Pull request merged**